src/Command/Command/Market/BookKeeper.php

```diff
@@ -127,21 +144,22 @@ protected function logException(\Exception $e, $logNewTrace = true) : void
      *
      * @return bool
      */
-    protected function isMaintenance() : bool
+    protected function isMaintenance(): bool
     {
-        return false;
-        $ch = \curl_init(\sprintf('https://%s', (new Config())->gemini->api->host));
+        if (!self::$maintenance) { // mostly for IDE to not yell
+            return false;
+        }
+        $ch = \curl_init(\sprintf('https://%s', Config::getInstance()->get('gemini')->api->host));
         \curl_setopt_array($ch, [
             CURLOPT_RETURNTRANSFER => true,   // return web page
             CURLOPT_HEADER         => false,  // don't return headers
             CURLOPT_FOLLOWLOCATION => false,  // follow redirects
             CURLOPT_MAXREDIRS      => 10,     // stop after 10 redirects
-            CURLOPT_ENCODING       => "",     // handle compressed
-            CURLOPT_USERAGENT      => "test", // name of client
+            CURLOPT_ENCODING       => '',     // handle compressed
+            CURLOPT_USERAGENT      => 'test', // name of client
```

Comment:
What's the point of adding this?

Reply:
Single ticks are the recommendation for string literals in php and even JS I believe.  It doesn't harm or better anything, just cleaner to standards pretty much